src/test/kotlin/com/igorwojda/string/ispalindrome/permutation/challenge.kt

```diff
@@ -4,7 +4,16 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun isPermutationPalindrome(str: String): Boolean {
-    TODO("not implemented")
+    var oneCharAllowed = false
```

Comment:
The name of this variable is a bit vague or counter-intuitive. Shouldn't this something like: oneCharAlreadyEncountered?